src/sentry/replays/utils.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Dict, Optional
```

Comment:
lifted from the profiles utils. since we're gonna delete this within a month or two, didn't feel worth it to try and DRY it.